65.md

```diff
@@ -4,73 +4,56 @@ NIP-65
+When seeking events **about** a user, where the user was tagged, Clients SHOULD use the READ relays of the user's `kind:10002` 
 
-Clients SHOULD presume that if their user has a pubkey in their ContactList (kind 3) that it is because they wish to see that author's feed-related events. But clients MAY presume otherwise.
+When broadcasting an event, Clients SHOULD:
 
-### Motivation
+- Broadcast the event to the WRITE relays of the author
+- Broadcast the event all READ relays of each tagged user. 
 
-There is a common nostr use case where users wish to follow the content produced by other users. This is evidenced by the implicit meaning of the Contact List in [NIP-02](02.md)
+## Motivation
 
-Because users don't often share the same sets of relays, ad-hoc solutions have arisen to get that content, but these solutions negatively impact scalability and decentralization:
+The old model of using a fixed relay list per user centralizes in large relay operators: 
 
-  - Most people are sending their posts to the same most popular relays in order to be more widely seen
-  - Many people are pulling from a large number of relays (including many duplicate events) in order to get more data
+  - Most users submit their posts to the same highly popular relays, aiming to achieve greater visibility among a broader audience.
+  - Many users are pulling events from a large number of relays in order to get more data at the expense of duplication
   - Events are being copied between relays, oftentimes to many different relays
+  
+This NIP allows Clients to connect directly with the most up-to-date relay set from each individual user, eliminating the need of broadcasting events to popular relays. 
 
-### Purposes
-
-The purpose of this NIP is to help clients find the events of the people they follow, to help tagged events get to the people tagged, and to help nostr scale better.
-
-### Suggestions
+## Final Considerations
 
-It is suggested that people spread their kind `10002` events to many relays, but write their normal feed-related events to a much smaller number of relays (between 2 to 6 relays). It is suggested that clients offer a way for users to spread their kind `10002` events to many more relays than they normally post to.
+1. Clients should guide users to keep `kind:10002` lists small (2-4 relays). 
 
-Authors may post events outside of the feed that they wish their followers to follow by posting them to relays outside of those listed in their "Relay List Metadata".  For example, an author may want to reply to someone without all of their followers watching.
+2. Clients should spread an author's `kind:10002` events to as many relays as viable, all of them if possible. 
```

Comment:
Not sure about "all of them."  I use 54 relays that seem to actually operate properly, among 475 entries in my list (some of which are bugs)

Reply:
Removed